webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -90,6 +92,18 @@ test.describe('Experiment List', () => {
       await tableFilter.close();
       await waitTableStable();
     });
+    await test.step('Reset Show Archived', async () => {
+      const tableFilter =
+        await projectDetailsPage.f_experimentList.tableActionBar.tableFilter.open();
+      if (
+        (await tableFilter.filterForm.showArchived.pwLocator.getAttribute('aria-checked')) ===
```

Comment:
this might want something similar as my other comment with the repeatwithfallback stuff